app/controllers/admin/problems_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
 
   def create
     authorize contest, :edit?
-
+    
```

Comment:
It is a bad practice to have trailing whitespace in the source code. It introduces such diffs in git, which doesn't actually change anything and pollute the changes. You can make sublime trim the trailing whilespace on save with this setting:

https://www.sublimetext.com/forum/viewtopic.php?f=3&t=3174